### PR TITLE
docs(input-schema-specs): add example pattern regex for full ISO date…

### PR DESCRIPTION
**Pull request by metalwarrior665**
See discussion: https://apify.slack.com/archives/C01E7CSJ2PP/p1734018697179259

Not sure if this is the best place to put it.

### CODE REVIEW

**Review by gippy — APPROVED** [review]
I think this is fine. Just a little unsure of the two notes under eachother.

**Comment by metalwarrior665** [comment]
Will wait for @TC-MO if he thinks we can do something better than second note

File: sources/platform/actors/development/actor_definition/input_schema/specification.md [hunk 1]
```diff
@@ -241,6 +241,12 @@ When using escape characters `\` for the regular expression in the `pattern` fie
 
 :::
 
+:::note Date with time
+
+While the `datepicker` editor doesn't support setting time value visually, you can allow users to pass it via JSON using `"pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])(T[0-2]\\d:[0-5]\\d(:[0-5]\\d)?(\\.\\d+)?Z?)?$|^(\\d+)\\s*(minute|hour|day|week|month|year)s?$"`. This regex allows users to optionally extend the date with full ISO datetime format or pass `hours` and `minutes` as a relative date. Make sure to explain to users in the description that the time values are expected to be in UTC.
+
+:::
+
```
Comment (TC-MO) [inline sources/platform/actors/development/actor_definition/input_schema/specification.md:249]: Maybe we can forego the second admonition completely and make it part of the prose?


```suggestion
#### Advanced date and time handling

While the `datepicker` editor doesn't support setting time values visually, you can allow users to handle more complex datetime formats and pass them via JSON. The following regex allows users to optionally extend the date with full ISO datetime format or pass `hours` and `minutes` as a relative date:

`"pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])(T[0-2]\\d:[0-5]\\d(:[0-5]\\d)?(\\.\\d+)?Z?)?$|^(\\d+)\\s*(minute|hour|day|week|month|year)s?$"`

When implementing time-based fields, make sure to explain to your users through the description that the time values should be provided in UTC. This helps prevent timezone-related issues.

```

**Review by TC-MO — COMMENTED** [review]
Proposed rewrites for the second admonition to prevent stacking admonitions next to each other

**Comment by metalwarrior665** [comment]
@TC-MO Good by you now?

**Review by TC-MO — APPROVED** [review]

**Pull request merged**